soul-admin/src/main/resources/application-local.yml

```diff
@@ -27,7 +27,7 @@ spring:
   datasource:
     url: jdbc:mysql://localhost:3306/soul?useUnicode=true&characterEncoding=utf-8
     username: root
-    password:
+    password: 1234
```

Comment:
why not modidy this?

Reply:
My bad, I change back to the original  value.